akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala

```diff
@@ -62,10 +62,10 @@ private[http] final class BodyPartParser(
     if (illegalHeaderWarnings) log.warning(errorInfo.withSummaryPrepended("Illegal multipart header").formatPretty)
   }
 
-  val in = Inlet[ByteString]("BodyPartParser.in")
```

Comment:
those are fine as is IMO. No need to go all crazy with those renames, as long as we can identify which port and class it serves it's purpose